docs/airflow/dags-maintenance.md

```diff
@@ -1,39 +1,70 @@
+It also executes the [realtime validator](https://github.com/MobilityData/gtfs-realtime-validator)
+against the RT files and writes the validation notices as JSONL to GCS for querying.
 
-1. [GTFS Schedule DAGs](gtfs-schedule-dags)
-1. [GTFS Realtime DAGs](gtfs-realtime-dags)
-1. [MST Payments DAGs](mst-payments-dags)
-1. [Transit Database DAGs](transit-database-dags)
-1. [Views DAGs](views-dags)
+The raw proto files themselves are fetched via [the archiver](https://github.com/cal-itp/data-infra/tree/main/services/gtfs-rt-archiver-v3).
 
-(gtfs-schedule-dags)=
-## GTFS Schedule - DAGs Maintenance
+### Deprecated DAGs
 
-### DAGs Overview
+The following DAGs are still listed in the Airflow UI even though they are **deprecated or indefinitely paused**. They never need to be re-run.
 
-### Common Issues
+* `check_data_freshness`
 
-### Backfilling
+## Task-level considerations
 
-(gtfs-realtime-dags)=
-## GTFS Realtime - DAGs Maintenance
+Some tasks have unique considerations, beyond the requirements of their overall DAG.
 
-### DAGs Overview
+* **`PodOperators`**: When restarting a failed `PodOperator` run, check the logs before restarting. If the logs show any indication that the prior run's pod was not killed (for example, if the logs cut off abruptly without showing an explicit task failure), you should check that the pod associated with the failed run task has in fact been killed before clearing or restarting the Airflow task. If you don't know how to check a pod status, please ask in the `#data-infra` channel on Slack before proceeding.
 
-The primary DAG relevant to GTFS Realtime data is [parse_and_validate_rt_v2](https://github.com/cal-itp/data-infra/tree/main/airflow/dags/parse_and_validate_rt_v2).
-This DAG parses RT files into JSONL and writes them to GCS for querying via BigQuery external tables.
-It also executes the [realtime validator](https://github.com/MobilityData/gtfs-realtime-validator)
-against the RT files and writes the validation notices as JSONL to GCS for querying.
-
-The raw proto files themselves are fetched via [the archiver](https://github.com/cal-itp/data-infra/tree/main/services/gtfs-rt-archiver-v3).
+## Backfilling from the command line
 
-### Backfilling
+From time-to-time some DAGs may need to be re-ran in order to populate new data.
 
-This DAG does not have any tasks that `depends_on_past`, so you should be able to
-clear and re-run tasks as needed.
+Subject to the considerations outlined abov, backfilling can be performed by clearing historical runs in the web interface, or via the CLI:
+```shell
+gcloud composer environments run calitp-airflow-prod --location=us-west2 backfill -- --start_date 2021-04-18 --end_date 2021-11-03 -x --reset_dagruns -y -t "gtfs_schedule_history_load" -i gtfs_loader
+```
 
-(mst-payments-dags)=
 ## MST Payments - DAGs Maintenance
 
 The ETL is currently a scheduled Google Data Transfer job that transfers all files to `gcs://littlepay-data-extract-prod`
```

Comment:
Should this section be renamed -- is this specific to MST? Or should this be general payments? (At first I thought this was the job that stages data for direct consumption by whichever agency does that)

Reply:
Yeah it's general payments; I will rename (was trying to avoid changing since it's v1)